docs/source/driver-framework/drivers-overview.rst

```diff
@@ -51,18 +49,18 @@ The diagram features several entities that comprise the platform and its connect
   message bus is built around existing message bus software; currently VOLTTRON supports RabbitMQ and ZeroMQ. The
   VOLTTRON integration includes Pub/Sub and JSON RPC interfaces for agent and driver communication.
 * VOLTTRON Platform Agents and Subsystems - These agents and subsystems are installed on the platform to manage the
-  platform. They provide many user facing functions, aid in communication and manage other agents and drivers.
-* User's Agents - These agents are either agents included in the core repository but installed by a user, or user built
+  platform. They provide many user facing functions, aid in communication, and manage other agents and drivers.
+* User's Agents - These agents are either agents included in the core repository but installed by a user or user-built
   agent modules. They may perform a huge variety of user specified tasks, including data collection, device control,
   simulation, etc.
 * Platform Driver Agent - This agent is installed by a user to facilitate communication with drivers. Drivers should not
   communicated with directly - the platform driver implements several features for communicating with drivers to ensure
   smooth operation and consistent driver behavior.
-* Actuator agent - This agent is installed by user to provide scheduling capability for controlling drivers. The
+* Actuator agent - This agent is installed by a user to provide scheduling capability for controlling drivers. The
   Platform Driver does not include protections for race conditions, etc. It is always recommended to use the Actuator
   agent to set values on a device.
 * Device Driver - Drivers are special purpose agents which provide an interface between the platform driver and devices
-  such as Modbus, and BACnet devices. Drivers implement a specific set of features for protecting device communication
+  such as Modbus and BACnet devices. Drivers implement a specific set of features for protecting device communication,
```

Comment:
the comma doesn't make sense here...perhaps and?  Not even sure I would call a driver an agent...

Reply:
I initially put the the comma there because the participle phrase "ensuring uniform behaviors..." modifies the "Drivers" rather than the noun "specific set of features", thereby requiring a comma. But now that I think about it, I like the use of 'and' instead. Less commas and more clear.